terraform/provisioning/common.bash.tmpl

```diff
@@ -42,10 +42,10 @@ common::contained.af_run() {
   sudo docker run -d \
     --name contained.af \
     --network host \
-    quay.io/kinvolk/contained.af:container-escape-bounty-latest \
+    quay.io/kinvolk/contained.af:${contained_af_image_tag} \
```

Comment:
I would use `${contained_af_image}` instead of just the tag, and use the default value of `quay.io/kinvolk/contained.af:container-escape-bounty-latest`, so developers outside of Kinvolk could easily try their own images hosted at a different repository than `quay.io/kinvolk/contained.af`.